integration/cursors/awaitdata_test.go

```diff
@@ -308,6 +308,122 @@ func TestCursorsTailableAwaitData(t *testing.T) {
 	})
 }
 
+func TestCursorsTailableAwaitDataAfterInsertStress(t *testing.T) {
+	if !setup.IsMongoDB(t) {
+		t.Skip("https://github.com/FerretDB/FerretDB/pull/3961")
+	}
+
+	var count atomic.Int32
+
+	teststress.Stress(t, func(ready chan<- struct{}, start <-chan struct{}) {
+		testID := count.Add(1)
+
+		t.Run(fmt.Sprint(testID), func(t *testing.T) {
+			s := setup.SetupWithOpts(t, &setup.SetupOpts{
+				DatabaseName: fmt.Sprintf("%s_%d", testutil.DatabaseName(t), testID),
+			})
+
+			db, ctx := s.Collection.Database(), s.Ctx
+
+			collName := testutil.CollectionName(t)
+
+			opts := options.CreateCollection().SetCapped(true).SetSizeInBytes(10000)
+			err := db.CreateCollection(s.Ctx, collName, opts)
+			require.NoError(t, err)
+
+			collection := db.Collection(collName)
+
+			bsonArr, arr := integration.GenerateDocuments(0, 3)
+
+			_, err = collection.InsertMany(ctx, bsonArr)
+			require.NoError(t, err)
+
+			var cursorID any
+
+			cmd := bson.D{
+				{"find", collection.Name()},
+				{"batchSize", 1},
+				{"tailable", true},
+				{"awaitData", true},
+			}
+
+			ready <- struct{}{}
+			<-start
+
+			var res bson.D
+			err = collection.Database().RunCommand(ctx, cmd).Decode(&res)
+			require.NoError(t, err)
+
+			var firstBatch *types.Array
+			firstBatch, cursorID = getFirstBatch(t, res)
```

Comment:
```suggestion
			var firstBatch *types.Array
			firstBatch, cursorID := getFirstBatch(t, res)
```

There is no need to define `cursorID` above either.

(I wonder why everyone suddenly started using `var` instead of short variables assignments…)